src/components/Window.js

```diff
@@ -18,17 +18,20 @@ export class Window extends Component {
    * additional functionality based on workspace type
    */
   wrappedTopBar() {
-    const { manifest, window, workspaceType } = this.props;
+    const {
+      manifest, window, workspaceType, windowDraggable,
+    } = this.props;
     const { mosaicWindowActions } = this.context;
     const topBar = (
       <div>
         <WindowTopBar
           windowId={window.id}
           manifest={manifest}
+          windowDraggable={windowDraggable}
         />
       </div>
     );
-    if (workspaceType === 'mosaic' && window.maximized === false) {
+    if (workspaceType === 'mosaic' && windowDraggable) {
```

Comment:
This was going to be a bit cleaner until I remembered/realized that we're using two different drag-n-drop libraries for Mosaic and Elastic. _sigh_